### ISSUE TITLE
Is vivado indirectly required?

### ISSUE THREAD

**Issue by Kreijstal** [daveshah1/nextpnr-xilinx#82]
Since this depends on project x rays, and project x rays depends on an old version of vivado, I guess this project requires vivado, then?

**Comment by marzoul** [comment]
No. Project Xray only uses Vivado to document the bitstream format.
Nextpnr only uses the resulting document.

**Comment by Kreijstal** [comment]
ah, so step 1 of 
https://github.com/f4pga/prjxray
is optional?
Or it is for mantainers, not for users, right?

Edit: I actually don't know which steps are meant for users

**Comment by lehaifeng000** [comment]
vivado is optional
